thirtyinch/src/main/java/net/grandcentrix/thirtyinch/internal/TiActivityDelegate.java

```diff
@@ -211,8 +211,9 @@ public void onDestroy_afterSuper() {
                     "presenter configured as not retaining, destroying " + mPresenter);
         }
 
-        if (!destroyPresenter &&
-                !config.useStaticSaviorToRetain()
+        if (!destroyPresenter
+                && !config.useStaticSaviorToRetain()
+                && !mTiActivity.isActivityChangingConfigurations()
```

Comment:
The fix itself was just this `&& !mTiActivity.isActivityChangingConfigurations()` line.